### PR TITLE
fix some bug of chat content

### PR DESCRIPTION
**Pull request by ManhNTX**
https://user-images.githubusercontent.com/107173849/229983790-cdc548c3-ea8b-4658-b3a2-1f57d70fec71.mov

### CODE REVIEW

**Review by imGok — APPROVED** [review]

File: lib/pages/chat/events/message.dart [hunk 1]
```diff
@@ -52,348 +50,400 @@ class Message extends StatelessWidget {
...
-                        },
+        final rowChildren = <Widget>[
+          sameSender || ownMessage
+              ? SizedBox(
+                  width: Avatar.defaultSize,
+                  child: Padding(
+                    padding: const EdgeInsets.only(top: 8.0),
+                    child: Center(
+                      child: SizedBox(
+                        width: 16 * AppConfig.bubbleSizeFactor,
```
Comment (hoangdat) [inline lib/pages/chat/events/message.dart:126]: please define the style file for this widget

File: lib/pages/chat/events/message.dart [hunk 2]
```diff
@@ -52,348 +50,400 @@ class Message extends StatelessWidget {
...
+                        height: 16 * AppConfig.bubbleSizeFactor,
```
Comment (hoangdat) [inline lib/pages/chat/events/message.dart:127]: idem

File: lib/pages/chat/events/message.dart [hunk 3]
```diff
@@ -52,348 +50,400 @@ class Message extends StatelessWidget {
...
-                            horizontal: 12 * AppConfig.bubbleSizeFactor,
+                  ),
+                )
+              : FutureBuilder<User?>(
+                  future: event.fetchSenderUser(),
+                  builder: (context, snapshot) {
+                    final user =
+                        snapshot.data ?? event.senderFromMemoryOrFallback;
+                    return Avatar(
+                      size: 36,
```
Comment (hoangdat) [inline lib/pages/chat/events/message.dart:141]: idem

File: lib/pages/chat/events/message.dart [hunk 4]
```diff
@@ -52,348 +50,400 @@ class Message extends StatelessWidget {
...
+                  },
+                ),
+          Expanded(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                if (!sameSender)
+                  Padding(
+                    padding: const EdgeInsets.only(left: 8.0, bottom: 4),
```
Comment (hoangdat) [inline lib/pages/chat/events/message.dart:155]: idem

File: lib/pages/chat/events/message.dart [hunk 5]
```diff
@@ -52,348 +50,400 @@ class Message extends StatelessWidget {
...
+                        : FutureBuilder<User?>(
+                            future: event.fetchSenderUser(),
+                            builder: (context, snapshot) {
+                              final displayname =
+                                  snapshot.data?.calcDisplayname() ??
+                                      event.senderFromMemoryOrFallback
+                                          .calcDisplayname();
+                              return Text(
+                                displayname,
+                                style: TextStyle(
```
Comment (hoangdat) [inline lib/pages/chat/events/message.dart:167]: change style to M3 text style

**Review by hoangdat — COMMENTED** [review]
a lot of styles, dimensions need to move to style file

**Review by hoangdat — APPROVED** [review]

**Pull request merged**